src/components/App.tsx

```diff
@@ -27,9 +52,72 @@ const App: React.FC = () => {
           <Grid.Column stretched width={12}>
             <Segment>
               <Switch>
-                <Route path="/" exact component={Home} />
-                <Route path="/form" component={Register} />
-                <Route path="/applications" component={ApplicationsList} />
+                <Route exact path="/">
+                  <Home />
+                </Route>
+                <Route exact path="/login">
+                  <Login />
+                </Route>
+                <Route exact path="/applications">
+                  <ApplicationsList />
+                </Route>
+                <Route exact path="/applications/new">
+                  <ApplicationNew />
+                </Route>
+                <Route exact path="/applications/:appId">
+                  <Application />
+                </Route>
+                <Route exact path="/applications/:appId/:sectionName/page:page">
+                  <Application />
+                </Route>
+                <Route exact path="/applications/:appId/summary">
+                  <Application summary={true} />
+                </Route>
+                <Route exact path="/applications/:appId/approval">
+                  <Approval />
+                </Route>
+                <Route exact path="/admin">
+                  <Admin />
+                </Route>
+                <Route exact path="/admin/templates">
+                  <TemplateList />
+                </Route>
+                <Route exact path="/admin/templates/new">
+                  <TemplateNew />
+                </Route>
+                <Route exact path="/admin/templates/:templateId/:step">
+                  <Template />
+                </Route>
+                <Route exact path="/admin/users">
+                  <AdminUsers />
+                </Route>
+                <Route exact path="/admin/permissions">
+                  <AdminPermissions />
+                </Route>
+                <Route exact path="/admin/config">
+                  <Config />
+                </Route>
+                <Route exact path="/account">
+                  <Account />
+                </Route>
+                <Route exact path="/organisation/:orgName">
+                  <Organisation />
+                </Route>
+                <Route exact path="/organisation/:orgName/members">
+                  <OrgMemberEdit />
+                </Route>
+                <Route exact path="/notifications">
+                  <NotificationsList />
+                </Route>
+                <Route exact path="/notifications/:notificationId">
+                  <Notification />
+                </Route>
+                <Route exact path="/products">
+                  <ProductList />
+                </Route>
+                <Route exact path="/products/:productId">
+                  <Product />
+                </Route>
                 <Route component={NoMatch} />
```

Comment:
```suggestion
                <Route>
                  <NoMatch />
                </Route>
```

Just to keep consistent with other routes